xml/tuning_cgroups.xml

```diff
@@ -542,6 +505,33 @@ arises, it points out to improperly provisioned memory (or memory limits).
 </listitem>
 </itemizedlist>
 </sect2>
+<sect2 xml:id="sec-cgroups-user-sessions">
+<title>Resource control in user sessions</title>
+<para>
+ In order to apply cgroup resource control within user sessions,
+ controllers must be delegated user instances of &systemd;.
+ &productname; ships &systemd; default configuration that delegates <emphasis>no</emphasis> controllers.
+</para>
+<para>
+  You can use drop-in files to change the set of delegated controllers. For instance
+  <filename>/etc/systemd/system/user@.service.d/60-delegate.conf</filename> will add controllers to all users and
+  <filename>/etc/systemd/system/user@<replaceable>uid</replaceable>.service.d/60-delegate.conf</filename> will add it only to particular user. The content of the file should be like following:
```

Comment:
```suggestion
  You can use drop-in files to change the set of delegated controllers. For instance,
  <filename>/etc/systemd/system/user@.service.d/60-delegate.conf</filename> 
  adds controllers to all users, while 
 <filename>/etc/systemd/system/user@<replaceable>uid</replaceable>.service.d/60-delegate.conf</filename> adds a controller only to a particular user. The content of the file should be as follows:
```

Reply:
I notice incorrect singular propagated to your mod, I'd stack on top yours:
```diff
-60-delegate.conf</filename> adds a controller only to a particular user.
+60-delegate.conf</filename> adds controllers only to a particular user.
```